packages/graphics/bcm2835-driver/package.mk

```diff
@@ -43,53 +43,29 @@ make_target() {
     cp -PRv $FLOAT/opt/vc/include/* $SYSROOT_PREFIX/usr/include
 
   mkdir -p $SYSROOT_PREFIX/usr/lib
-    cp -PRv $FLOAT/opt/vc/lib/libbrcmEGL.so $SYSROOT_PREFIX/usr/lib
+    for f in $FLOAT/opt/vc/lib/*.so; do
```

Comment:
There is one sym link, `libGLESv1_CM.so -> libGLESv2.so`, that I thought it best to avoid. It probably makes no difference if we include it. I'll update the PR in a sec.